cdap-docs/_common/common-build.sh

```diff
@@ -358,7 +377,63 @@ function display_version() {
   echo "PROJECT_SHORT_VERSION: ${PROJECT_SHORT_VERSION}"
   echo "GIT_BRANCH: ${GIT_BRANCH}"
   echo "GIT_BRANCH_TYPE: ${GIT_BRANCH_TYPE}"
-  echo ""
+  echo "GIT_BRANCH_PARENT: ${GIT_BRANCH_PARENT}"
+}
+
+function set_messages_file() {
+  TMP_MESSAGES_FILE="/tmp/$(basename $0).$$.tmp"
+  export TMP_MESSAGES_FILE
+}
+
+function cleanup_messages_file() {
+  if [[ "x${TMP_MESSAGES_FILE}" != "x" && -a ${TMP_MESSAGES_FILE} ]]; then
+    rm -f TMP_MESSAGES_FILE
+  fi
+}
+
+function clear_messages() {
+  MESSAGES=""
+  set_messages_file
+}
+
+function set_message() {
+  if [ "x${MESSAGES}" == "x" ]; then
+    MESSAGES="${1}"
```

Comment:
Sure. Change this line to `MESSAGES=${*}` and it'll take all arguments to `set_message`, not just the first one. You'll need to do the same 2 lines below.

Now, you can `set_message "some message"` **or** `set_message some message` and they'll be treated the same. Otherwise, you **must** enclose the arguments to `set_message` in quotes, when it's not really necessary.